web/src/test/java/ua/vertex/academy/service/RouteServiceImplTest.java

```diff
@@ -77,16 +78,18 @@ public void update() throws Exception {
         TEST_ROUTE.setName("testUpdate");
 
         List<WayPoint> wayPoints = new ArrayList<>();
-        WayPoint first = WayPoint.newBuilder().setRouteId(ROUTE_ID).setX(1).setY(1).setHeight(1).setAccuracy(1).build();
-        WayPoint second = WayPoint.newBuilder().setRouteId(ROUTE_ID).setX(3).setY(2).setHeight(2).setAccuracy(2).build();
-        WayPoint third = WayPoint.newBuilder().setRouteId(ROUTE_ID).setX(3).setY(3).setHeight(3).setAccuracy(3).build();
+        WayPoint first = WayPoint.newBuilder().setId(ThreadLocalRandom.current().nextLong())
```

Comment:
Too ... much ... builders .... need .... be ... strong..... use .... force.... Luke....